opentracing-spring-web-starter/pom.xml

```diff
@@ -24,6 +24,11 @@
       <artifactId>spring-webmvc</artifactId>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-webflux</artifactId>
+      <scope>provided</scope>
```

Comment:
I think this should be optional? We don't want users to depend on webflux

Reply:
Thanks, `optional` makes more sense than `provided`.